config/sync/core.entity_view_display.node.page.default.yml

```diff
@@ -4,7 +4,10 @@ status: true
 dependencies:
   config:
     - field.field.node.page.field_branch
+    - field.field.node.page.field_display_titles
+    - field.field.node.page.field_hero_title
     - field.field.node.page.field_paragraphs
+    - field.field.node.page.field_subtitle
```

Comment:
I wonder why you went with the approach with adding a separate title field and a subtitle field.

It might be good but I cannot tell whether this is what was actually requested. Personally I had expected a single toggle field.

Reply:
I'm not sure what you mean by "seperate"

However, maybe this answers your question:

The standard label field on a node, is still used by the `<title>` tag, and the admin content search.
For example, it might make sense to have the frontpage named "frontpage", but you will probably not want to show that on the actual page.

As for the subtitle - the design has the subtitle as part of it, and we already had a "field_subtitle" used on articles, so i figured i'd re-use it, so we can also use it in teasers and what not

<img width="574" alt="Screenshot 2024-02-27 at 13 44 18" src="https://github.com/danskernesdigitalebibliotek/dpl-cms/assets/12376583/0849a310-b1d4-4430-ad76-d3dd81a1691e">

Reply:
> The standard label field on a node, is still used by the <title> tag, and the admin content search.
For example, it might make sense to have the frontpage named "frontpage", but you will probably not want to show that on the actual page.

Separate as in separate from the standard title field.

I can understand that people might want to separate the administration title from the displayed title but the fact that with this change the administration title is still used in the `<title>` tag means that it is still not totally hidden to users.

To me it would be more consistent to tell users to use titles that they actually want to show to end users whether it is (partially) hidden from view or not.

Having alternate titles will cause complexity in other situations such e.g. user facing views.

> As for the subtitle - the design has the subtitle as part of it, and we already had a "field_subtitle" used on articles, so i figured i'd re-use it, so we can also use it in teasers and what not

Personally I had imagined the subtitle being a paragraph. Perhaps that not good enough.

I understand your point about subtitles in lists but with this approach I think that is somewhat confusing that the only way you can get a subtitle into lists is by enabling the "Display titles" option.